keras_cv/layers/preprocessing/random_shear.py

```diff
@@ -151,3 +170,171 @@ def get_config(self):
             }
         )
         return config
+
+    def augment_bounding_boxes(self, bounding_boxes, transformation, **kwargs):
+        image = None
+        image = kwargs.get("image")
+        if self.bounding_box_format is None:
+            raise ValueError(
+                "`RandomShear()` was called with bounding boxes,"
+                "but no `bounding_box_format` was specified in the constructor."
+                "Please specify a bounding box format in the constructor. i.e."
+                "`RandomShear(bounding_box_format='xyxy')`"
+            )
+        height, width, _ = image.shape
+        image = tf.expand_dims(image, axis=0)
+
+        bounding_boxes = keras_cv.bounding_box.convert_format(
```

Comment:
we can actually rely on unbatched image inputs after https://github.com/keras-team/keras-cv/pull/556

Reply:
feel free to rebase and rely on this to avoid the tf.expand_dims() call on image.